### PR TITLE
Update Consumption Unit to Watt-hour to Match In-Car Unit

### PR DESCRIPTION
**Pull request by mattw01**
Updated Consumption unit in Drives and Efficiency dashboards to Watt-hour (Wh) per #675

### CODE REVIEW

File: grafana/dashboards/drives.json [hunk 1]
```diff
@@ -578,7 +578,7 @@
...
...ER BY start_date DESC\n)\nSELECT\n    start_date_ts,\n    end_date_ts,\n    car_id,\n    start_path,\n    end_path,\n    duration_str,\n    drive_id,\n    -- Columns\n    start_date as start_date_[[length_unit]],\n    start_address,\n    end_address,\n    duration_min,\n    convert_km(distance, '$length_unit') AS distance_$length_unit,\n    start_battery_level as \"% Start\",\n    end_battery_level as \"% End\",\n    convert_celsius(outside_temp_avg, '$temp_unit') AS outside_temp_$temp_unit,\n    convert_km(avg_speed, '$length_unit') AS speed_avg_$length_unit,\n    power_max,\n    reduced_range as has_reduced_range,\n    range_diff * car_efficiency as \"consumption_kWh\",\n    CASE WHEN is_sufficiently_precise THEN range_diff * car_efficiency / distance * 1000 * CASE WHEN '$length_unit' = 'km' THEN 1\n                                                                                                WHEN '$length_unit' = 'mi' THEN 1.60934\n                                                                                           END\n    END AS consumption_kWh_100,\n    CASE WHEN is_sufficiently_precise THEN distance / range_diff\n         ELSE NULL\n    END AS efficiency\nFROM data;",
```
Comment (adriankumpf) [inline grafana/dashboards/drives.json:581]: In 563cf7832a2845daaa17d810924989140df32154 the `reduced_range` field was slightly modified. These changes are not reflected here. Could you please rebase/merge your branch to pull in this commit?

File: grafana/dashboards/efficiency.json [hunk 2]
```diff
@@ -181,8 +181,8 @@
         "#d44a3a"
       ],
       "datasource": "TeslaMate",
-      "decimals": 1,
-      "format": "kwatth",
+      "decimals": 0,
+      "format": "watth",
```
Comment (adriankumpf) [inline grafana/dashboards/efficiency.json:185]: The `/ 100km` (`/ 100mi`) postfix value also needs to be changed for both consumption panels :)

File: grafana/dashboards/drives.json (hunk 1 above)
Comment (adriankumpf) [reply grafana/dashboards/drives.json:581]: The Trip dashboard has a minimal "Drives" panel that shares most fields with this table. I think it makes sense to update the consumption there as well.

Comment (mattw01) [reply grafana/dashboards/drives.json]: Ah, I was testing with the last release version and forgot to merge in the latest commits. Thanks for catching that. Also updated the Trip dashboard as well.

**Comment by mattw01** [comment]
Modified it a bit more to include the length unit.
Grafana supports custom units since version 6.6. I updated the units to include the current $length_unit. So it should display either Wh/km or Wh/mi to be a bit more specific.

File: grafana/dashboards/efficiency.json (hunk 2 above)
Comment (mattw01) [reply grafana/dashboards/efficiency.json:185]: Fixed these panels, thanks!

File: grafana/dashboards/efficiency.json [hunk 3]
```diff
@@ -477,12 +477,12 @@
...
           ],
           "dateFormat": "YYYY-MM-DD HH:mm:ss",
-          "decimals": 2,
+          "decimals": 0,
           "mappingType": 1,
           "pattern": "consumption_km",
           "thresholds": [],
           "type": "number",
-          "unit": "kwatth"
+          "unit": "Wh/km"
```
Comment (adriankumpf) [inline grafana/dashboards/efficiency.json:485]: Nice! 👍

**Pull request merged**